assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -161,10 +162,23 @@ export const selectors = {
 	 * @param {Object} state Data store's state.
```

Comment:
The new parameters should be documented here.

Reply:
Oh yeah, thanks!